notebooks/07_Experiment_Tracking/07_01_Metadata.ipynb

```diff
@@ -490,9 +649,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.7"
+   "version": "3.7.6"
```

Comment:
which notebook image are you using? Trying to see the version difference

Reply:
I'm using: `gcr.io/kubeflow-images-public/tensorflow-1.15.2-notebook-cpu:1.0.0`, not sure why the version changes.

Reply:
Okay, will fixed in the next version.